hooks/useCreateList.ts

```diff
@@ -1,13 +1,42 @@
-import { createContext, useContext, useEffect, useState } from 'react'
+import { BaseSyntheticEvent, createContext, useContext, useEffect, useState } from 'react'
 import { useQuery } from 'react-query'
 import { getAllServices } from 'api'
 import { Service } from 'models'
+import { useDisclosure } from '@chakra-ui/react'
+import { yupResolver } from '@hookform/resolvers/yup'
+import { SubmitHandler, useForm, UseFormReturn } from 'react-hook-form'
+import * as yup from 'yup'
+
+export interface CreateListForm {
+  name: string
+  creator: string
+  description: string
+}
+
+const fieldRequiredMessage = 'Cannot be blank'
+
+const createListSchema = yup.object({
+  name: yup.string().required(fieldRequiredMessage),
+  creator: yup.string().required(fieldRequiredMessage),
+  description: yup.string().required(fieldRequiredMessage),
```

Comment:
Just want to make sure, is this where the list is created? Don't see where we add the services ids

Reply:
So this schema is just for the form fields, we'll grab service ids from the `selectedFields` state below. Once we merge your PR in I can integrate the create API call with the form.